src/HSE/Util.hs

```diff
@@ -1,30 +1,48 @@
 
 module HSE.Util(
     extensionImpliedBy,
+    extensionImpliedEnabledBy,
+    extensionImpliedDisabledBy,
     extensionImplies,
     ) where
 
+import GHC.Util.Outputable
+import GHC.LanguageExtensions.Type
+import Data.Maybe
 import qualified Data.Map as Map
 import qualified Language.Haskell.GhclibParserEx.DynFlags as GhclibParserEx
-import Language.Haskell.Exts(Extension(..), parseExtension)
 
+-- Because we don't have Ord or Data.Data.Data we fall back on String
```

Comment:
No `Ord` so extensions can't be keys, no `Data` so that rules out using `HsExtendInstances` which was my first idea.